app_config/v2g_liberty_dashboard.yaml

```diff
@@ -327,35 +349,18 @@ views:
                 state: "on"
             card:
               type: markdown
-              content: |-
-                The car is not connected.
-                To start charging, please connect the car to the charger.
-              title: Charging not possible
-              card_mod:
-                style: |
-                  ha-card {
-                    background-color: #FEF0EA;
-                    color: #000000;
-                    border: solid 2px #E14D0E;
-                  }
+              content: >-
+                <ha-alert alert-type="warning" title="No car is connected">To start Max-charge-now, please connect the car to the charger.</ha-alert>
           - type: conditional
             conditions:
               - entity: binary_sensor.is_flexmeasures_live
                 state: "off"
             card:
               type: markdown
               content: >-
-                It seems the FlexMeasures server is offline. It is not possible
-                to get an automated charging schedule. Set charging manually.
-                Please check with system administrator.
-              title: Server offline
-              card_mod:
-                style: |
-                  ha-card {
-                    background-color: #FEF0EA;
-                    color: #000000;
-                    border: solid 2px #E14D0E;
-                  }
+                <ha-alert alert-type="error" title="FlexMeasures seems to be offline">It now is not possible to get an automated charging schedule.</ha-alert>
+                Usually this problem is solved in a few minutes. If it remains over a longer period, please set charging 
+                manually and check with system administrator.
```

Comment:
Are these still visible even though they are defined outside of the ha-alert tag?